A/ASL/build_tarballs.jl

```diff
@@ -26,36 +25,26 @@ cflags=""
 if [ -f $WORKSPACE/srcdir/asl-extra/arith.h.$target ]; then
     cp $WORKSPACE/srcdir/asl-extra/arith.h.$target ./arith.h
 fi
-if [ $target = "arm-linux-musleabihf" ]; then
+
+if [[ "${target}" == "arm-linux-musleabihf" ]]; then
     cp $WORKSPACE/srcdir/asl-extra/arith.h.arm-linux-gnueabihf ./arith.h
-fi
-if [ $target = "i686-linux-musl" ]; then
+elif [[ $target == "i686-linux-musl" ]; then
     cp $WORKSPACE/srcdir/asl-extra/arith.h.i686-linux-gnu ./arith.h
-fi
-if [ $target = "aarch64-linux-musl" ]; then
+elif [[ "${target}" == "aarch64-linux-musl" ]]; then
     cp $WORKSPACE/srcdir/asl-extra/arith.h.aarch64-linux-gnu ./arith.h
-fi
-if [ $target = "x86_64-unknown-freebsd11.1" ]; then
+elif [[ "${target}" == *-freebsd* ]]; then
     cp $WORKSPACE/srcdir/asl-extra/arith.h.x86_64-linux-gnu ./arith.h
     cflags="-D__XSI_VISIBLE=1"
-fi
-
-if [ $target = "x86_64-w64-mingw32" || $target = "i686-w64-mingw32" ]; then
```

Comment:
This syntax is wrong:
```
sandbox:${WORKSPACE} # [ $target = "x86_64-w64-mingw32" || $target = "i686-w64-mingw32" ]
bash: [: missing `]'
bash: x86_64-unknown-freebsd11.1: command not found
```
I'm surprised that this didn't make builders to fail

Reply:
Gee, you're right. It never complained locally either. Many thanks for looking into this!

Reply:
Actually in the log I could see the error, but that didn't make the build to fail, which is strange :confused:

Reply:
At any rate, I confirm that your changes also work for me locally.